api/config/token.ts

```diff
@@ -1,14 +1,24 @@
-const jwt = require("jsonwebtoken");
+import jwt, { JwtPayload } from "jsonwebtoken";
 
-const SECRET = process.env.SECRET_PASSWORD;
+const SECRET = process.env.SECRET_PASSWORD || "";
 
-function generateToken(payload: string) {
-  const token = jwt.sign({ payload }, SECRET);
-  return token;
+export interface UserPayload extends JwtPayload {
+  id: number;
```

Comment:
It would be awesome to derive this type from User type.
Something like:
```
interface User = {
   _id: string
   name: string
   lastName: string
   email:string
   image:string
}

type UserJWT = Pick<User, '_id' | 'email'>

export interface UserPayload extends JwtPayload, UserJWT
export type UserPayload = JwtPayload & UserJWT
```